pyproject.toml

```diff
@@ -6,6 +6,7 @@ requires = [
     "decorator",
     "pyyaml",
     "scipy",
+    "einops",
```

Comment:
as we discussed before. einops is required by vllm already. No need to add it for vllm-ascend